src/ext.rs

```diff
@@ -1,8 +1,11 @@
 //! Safe wrapper around externalities invokes.
 
-use hash::{H256, Address};
-use uint::U256;
-use pwasm_std;
+use pwasm_std::{
+	self,
+	types::{H160, H256, U256}
+};
+
+type Address = H160;
```

Comment:
Why not export this type from `pwas-std`? Seems like it's used in plenty of places.

Reply:
The current problem is that in `pwasm-std` version `0.11` I decided to make `H160` and `Address` distinct types which broke lots and lots of code and I regret this decision now and will revert it for version `0.12`. So for the moment this is the quick fix.

Reply:
So when depending on `pwasm-std` version `0.12` the only thing that has to be changed are these few lines to something like that:

```rust
use pwasm_std::{
        self,
	types::{H160, H256, U256, Address}
};
```

Reply:
ah, cool